src/main/java/core/basesyntax/service/operationhandler/BalanceOperationHandler.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.service.operationhandler;
+
+import core.basesyntax.dao.StorageDao;
+
+public class BalanceOperationHandler implements OperationHandler {
+
+    @Override
+    public void makeOperation(String name, int quantity, StorageDao storageDao) {
```

Comment:
For all handlers - StorageDao should be a class field. Inject it using constructor.